src/assets/i18n/it.po

```diff
@@ -9834,6 +9963,16 @@ msgid ""
 "computers-servers-mchlp1140/\" target=\"_blank\">Apple documentation</a>  "
 "for more details."
 msgstr ""
+"I privilegi sono gli stessi dell'account ospite. L'accesso Ospite è "
+"disabilitato di default in Windows 10 versione 1709 e Windows Server "
+"versione 1903. E' richiesta un'ulteriore configurazione lato client per fornire "
+"l'accesso ospite a questi client. <br><br> <i>Client MacOS</i>: Tentando di "
+"accedere con un utente che non esiste in FreeNAS <i>non</i> connette in modo "
```

Comment:
tentando di accedere con un utente che non esiste in FreeNAS non connette - credo manchi il soggetto

Reply:
sì. confermo.
io metterei così:
_"Tentando di accedere con un utente inesistente in FreeNAS, l'utente <b>non</b> si connette in automatico come ospite."_ 
Mi pare che sia più corretto e rimane comprensibile.

Reply:
anzi: _"l'utente non si connette automaticamente come ospite."_